data/campaigns/Eastern_Invasion/terrain-graphics.cfg

```diff
@@ -0,0 +1,47 @@
+#textdomain wesnoth-ei
+# This file needs to be processed *after* all others in this directory
```

Comment:
Is this even true or was it copy-pasted from somewhere and isn't accurate here?

Reply:
Copy-pasted, not accurate here.